src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  return days < 3
+    ? days * dayCost
+    : days < 7
+      ? days * dayCost - threeDaysDiscount
+      : days * dayCost - sevenDaysDiscount;
```

Comment:
It's better to use ternary operator when you need to choose one of the values according to some condition. When you have such nesting, it's better to use if else statement